core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>ArduPilot Manager</title>
+    <link rel="stylesheet" href="/static/css/metro-all.css" />
+    <link rel="stylesheet" href="/style.css" />
+    <script src="/static/js/vue.js"></script>
+    <script src="/static/js/axios.min.js"></script>
+    <script src="/static/js/metro.js"></script>
+</head>
+
+<body>
```

Comment:
It would be nice to have the status of the mavlink manager, if it's running, restarting and etc.

Reply:
That's a good idea. We need some additions on the backend to make it work though.
Will open an issue for that. Definitely something urgent.

Reply:
Opened #229 to follow that.